spec/index.bs

```diff
@@ -7014,6 +6997,58 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
         or the third item of the sequence.
 </div>
 
+# Features Definitions # {#feature-definitions}
+
+The capabilities of each [=feature=] approved by the WebGPU Working Group are defined here.
+
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>depth-clamping</dfn> ## {#depth-clamping}
+
+Issue: Define functionality when the {{GPUFeatureName/"depth-clamping"}} [=feature=] is enabled.
+
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>depth24unorm-stencil8</dfn> ## {#depth24unorm-stencil8}
+
+Allows for explicit creation of textures of format {{GPUTextureFormat/"depth24unorm-stencil8"}}.
+When the {{GPUFeatureName/"texture-compression-bc"}} [=feature=] is enabled the following
```

Comment:
Oh, this was actually a copy/paste error on my part, I'll fix it. Though that raises a question for me: DO any of the features currently have interdependencies?

Also, reading though the text I've got here I wonder if the phrasing of "When the [feature] is enabled the following {{GPUTextureFormat}}s must be supported:" suggests that the feature _may_ be available otherwise at the discretion of the hardware/UA and passing the feature string merely enforces it. I think I'll try and reword it to avoid that possibility.